rpi23-gen-image.sh

```diff
@@ -350,7 +350,7 @@ if [ -r "/dev/mapping/${CRYPTFS_MAPPING}" ] ; then
   exit 1
 fi
 
-# To delete an old build
+# Don't clobber an old build
```

Comment:
- [ ] changing the comment to be inconsistent with the code inside the conditional ... is wrong.

Since I've already raised this in a previous review, and your response in b85e1ca9 did nothing about it, I'll explain further.

In your file `rpi23-gen-image.sh` is a comment "Don't clobber an old build", above a conditional check for the existence of the build directory, which executes `rm -rf` to delete it.  This is just simply inconsistent.  The comment doesn't say what the code does.  The comment is a lie.  We can't abide lies in code.

Reply:
I think I get your point now, will make some changes and get back to you.

Reply:
Should I change the comment to: "To check for the existence of an old build folder and delete it if it does"?

Reply:
You seem to be missing the history of change in this code.  You're thrashing around like a dying fish.

Originally upstream wrote this to require that you delete builds yourself.  This is a useful feature because it prevents losing valuable work.  It is easy to overcome.

In 2dd011a18b6932f6229f78fac8e449c11526344f and 646006f4604e3adf4962d2ec2d8006cb464b88e6 you decided to delete the build automatically; but you didn't change the comment.

My review at the time was based on the assumption that you were doing this change for yourself only (not me or @Hrishi1999), and _rather than change rpi23-gen-image.sh to clobber old builds, you might instead learn to use compound statements in shell, the ctrl+r key for recalling them, or small wrapper scripts,_  You chose not to respond to that review comment.

These changes eventually became a squash and rebase commit e52bb817a22a3e782798346c2839207e8dc8b57c which made the comment inconsistent with the code.

In response, I raised #5, rather than insist.

In c2ea0feb8161aa2a2d94b19e650b0c89e659f081 you changed the comment to _To delete an old build_.

Your pull request now seeks to change the comment from _To delete an old build_ to _Don't clobber an old build_.

And now you are asking if the comment should be changed again ... it is truly disappointing.

Please use the tools available.  Read the changes proposed by your pull request; the sections in red and green.  That's what anyone else will be reading; and if you don't understand why a change is proposed, perhaps you need to fix it before making the pull request.

Reply:
Sorry to disappoint you due to misconception on my side. So are you suggesting that I change that code and the comment to what was initially there in the upstream? Which won't delete the build folder automatically but will notify the user that it exists and it won't proceed further with building the image because of that.

Reply:
> are you suggesting that I change that code and the comment to what was initially there in the upstream?

I'm saying in this review, that _changing the comment to be inconsistent with the code inside the conditional_, which is what your pull request is proposing to do, _is wrong_.

Can you see that your pull request is asking for this change?

That's blocking this pull request.

The rest of what I said is illustrative of your misunderstanding.

It is up to you what to propose change, via commit to the `gsoc` branch assoicated with this pull request, but the change at the moment is wrong because it makes the comment lie about the code.  I've also said weeks ago your previous changes to this code are wrong as well, but that's not directly what this pull request is about.

You've also used "git push" to rewrite the history of the Sugar Labs repository, merging in all your changes to the `master` branch without review.  The last two commits used to be 667c8b080ab057444176d7717881a3d77082f25a and e52bb817a22a3e782798346c2839207e8dc8b57c but everything you did in your private repository is now merged with the history here.  This means that when someone clones the repository, they get a copy of every one of those activity bundle files which are then deleted, because they form part of history now.  What a waste.

Reply:
Alright, I've resetted that code to what was originally there in the upstream for now while I figure out a better solution for this.
Regarding merging changes to master, that happened by accident sorry about that. 
I'll go through the previous changes too and fix them within a couple of days.
I hope this PR can be merged now